### ISSUE TITLE
Article: Looking Back at the W3C Web Accessibility Initiative

### ISSUE THREAD

**Issue by CherylLouise808** [a11yproject/a11yproject.com#1077]
"The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and implement accessibility."

In a five hundred word piece, I want describe the background and history of the Web Accessibility Initiative,  its focus, and what the WAI offers users today, specifically content writers.

Access to digital information is key in today's modern technological work, study, and play environment. As a reference librarian I have been called on to assist users of different abilities with assessing information from computers. W3C Web Accessibility Initiative has helped transform otherwise challenging experiences into those which enhance all people's lives. But it wasn't always around.

As a retired health professional-turned-librarian and writer heading toward retirement (Yay!) I enjoy writing about health/wellness/medical issues and related topics. My work has appeared in NailPro Magazine and MedicalEconomics.com. Please check out www.cherylbranche.com for samples.

If you like the Looking Back at  the W3C Web Accessibility Initiative idea, I am happy to write the piece.

Sincerely,
Cheryl Branche

**Comment by CherylLouise808** [comment]
Hi Eric Bailey:

Expected deadline over shot. No way I can get this out by 11PM-end of today.

 Am trying to reach out to another early founder of Web
Accessibility Initiative and some of the early institutions that were
involved.

May I obtain an extension until Monday, September 28, 2020 at the risk of
ruining my deadline keeping reputation?

Cheryl

**Comment by ericwbailey** [comment]
Hey Cheryl, thanks for the head's up. That's totally fine by me, I'd rather have something you're happy with than meeting your own self-imposed deadline 🙂

**Comment by CherylLouise808** [comment]
Before the Magnifying Glass: A Quite Brief History of the Web
Accessibility Initiative by Cheryl L. Branche




None of the current tools of the web that make the web easier to use
started in a vacuum. Wheels started to turn to make the web more
accessible in 1999.


Before the magnifying glass, many web users had difficulty using the
the webpages. The World Wide Web COnsortium (W3C)'S Daniel
DArdeniller, supported by IBM and Ford Foundation (Check FC,
maintained a few pages on the WE.org website to address web
accessibility issues. In 1999, he with W3C created a single standard
for web content accessibility entitled Web Content Accessibility
Guidelines, (WCAG) that  met the needs of persons, organizations, and
NGOs, governments across the globe. It addressed the needs of users in
education and employment commerce, health care, government, and
recreation in addition to others and supported social inclusion for
people who are older, who live in rural areas, and who also live in
developing countries.

The term web content "refers to information in a web page or
application, including natural information such as text information
and sounds. And also, code and markup, that one finds a structure
presentation, etc." Source: Web Content Accessibility
Guidelines-overview(WCAG)". The name of the website is Ws three C.
(http://www.W3C.org/AWI/standards/-guidelines/WCAG/WAI/standards/-guidelinesWACG/0WC3web
initiative accessibility initiative


Web Accessibility refers to websites, tools and technologies, which
include design and development elements that enable people with
disabilities can perceive, understand, navigate and interact with the
web and contribute to the web. Web accessibility initiatives address:

Auditory disabilities, cognitive disabilities neurological
disabilities, physical disabilities speech disabilities and visual
disabilities. It addresses people who use digitally equipment with
small screens or different input modes. Older people with disabilities
that change due to aging, people with temporary disabilities such as
those with broken limbs or lost glasses. People with situational
limitations, such as in "bright light or an environment where they
cannot listen to audio," people who will using a slow internet
connection, or have a limited or expensive bandwidth.


Here is a list of videos (https://youtube.com/whoth?V=3f31oufqFS)that
may . It's called web accessibility perspectives compilation of 10
topics. There is also a business case for accessibility. In many
cases, web accessibility as required by law. resources will be
included at the end of our article. All right.


Web Accessibility INitiative from within the  World Wide Web
Consortium came into being in the fall of 1996, as a few like minded
people like Daniel dargan la began looking at how "to make the web
more accessible to people who would otherwise be unable to use the
internet." For nine months the project was organized. IBM and other
organizations have supported the development and inclusion of
webpages, to explain the importance of web accessibility and included
links and project references. January 6 1997 found pioneers of the
World Wide Web Consortium movement drafting a draft briefing package,
which described what "web accessibility" was, why it was important,
and plans to address it.


By February of 1997. The Web Accessibility Initiative, became known
widely as WAI is guided by five key words: colon, international
normative, consensus, predictability and participation. The main
challenges for the push for web accessibility was marketing and
obtaining corporate cooperation, integrating the features into
webpages and programs throughout the internet, providing education and
training to convert already established websites versions that were
web accessible to all.   A WAI international program office (IPO) was
created to reach out to technical experts required to make the web
more accessible. It managed programs and funding related to its goal.
International Business Machines IBM and Microsoft and other US funding
groups provided funding for the olearly initiative. With that support,
on April, in 1997, the Web Accessibility Initiative, officially
launched.


Almost four months later, an MIT meeting helped to secure integral
WAItion of web accessibility technology. Today, Web Accessibility
Initiative is supported by Adobe HP, IBM, and others as well as the US
Department of Health and Human Services. Most browsers support web
accessibility. The effort is far from complete additional webpages
apps and services must implement web accessibility standards that
level the playing fields for disabled. No, web page app services must
implement web accessibility standards period from the fall of 1996 to
the end. The reference used is the history of web accessibility. By
Elias it is men knowledge of the web site is Ai L dot L y slash
journal slash history hyphen web hyphen accessibility slash the
author's name is spelled capital i l i A s capital is ma n a l, you
know, Al ij EV. The website was accessed on September 26 2020.
According to W AI early days, an article written by Daniel Dodd Dale
yay capital D AR di Ll, er, I'm Mike paciello capital P a CI Ll o of
the URI Rubin ski insight foundation. I had been maintaining the web
accessibility for people with disabilities web pages. Um, on the World
Wide Web Consortium website. Those couple of pages included references
to other projects at trace, University of Toronto and I see add. Mr.
Dave racket and Doug cc three WWDC staff person was was largely
involved on the technical side. Donald, Daniel da da da was assigned
to the security area. and the first joint payment initiative at WC
three and began helping. Mike paciello. In 1996, um, the discussions
with Mike between Mike, Daniel and Jim Miller, who was the head of WC
three Technology and Society road domain proposed a membership
resulted in proposed membership to move forward with a real w three c
project and web in the web accessibility area, considering that there
was a lack of net of standard guidelines and a divergence of
individual initiatives that were not useful to programmers, or users.
They're not in 1996 September guideline of the W three c Consortium.
Tim berners Lee writes, or ass should w three C or resources be spent
on providing support for people who have disabilities and others with
challenges with accessing the worldwide web. And second question was,
would your organization ask of the members, possibly be prepared to
provide effort or funds, or to match funds from other sponsors this
question should be in quotes. And these two questions were asked of
World Wide Web Consortium members, and were asked to provide a
response. Daniel Dunn in LA. I had had some experience with computers
accessibility. Using Unix. Excellent. x t motif, and D AC x and ice
and the Mercator project. Banana calm, with Bill Walker and Donna
converse in January 6 of 1999 white house representatives from
academia, industry, funding agencies, see List of funding agents or
see List of representatives presented a draft briefing package WCC was
designated as the ideal host for the proactive program in the area of
web accessibility link to the draft briefing packages included. On
January, 15 through the 16th of 1997 of the WCC advisability committee
meeting, Jim Miller presented the plan. As an upcoming bullet in the W
three C with external funding. The project was meant to include users,
and members and non members of a W three C. In quote breeding package
to being prepared by Daniel garden gay and Mick paciello from the
Rubin ski insight foundation should be available by the end of
February, my house meeting provides framework for an international
project office with DNS from a combination of WC three existing
resources, u s and VC funding and additional industry contributions.
An estimated total cost is $1.3 billion dollars per year for three
years close quote the bankruptcy was created in February of 1997,
providing resources on education and tools for content providers, was
a challenge for the consortium since it was mostly working with
software developers. So the whi international program office was
created to manage the extra funding, and the new activity. In April of
1997, the web conference in Santa Clara hosted the official launch of
W AI is all w three C has already received financial support and
endorsement funding agencies. By that time. In May, 1997, Daniel
Daniele organized the first series of wi technical F, as in Frank to F
meetings in Sofia and T bolus in France. In May, the first w AI public
list was launched. So, the mean in WC fury, and Sophia, and Sophia
antipolis France was convened to work on the technical issues at stake
in the world. In the WCC consortium web accessory. Accessibility
Initiative and included pot support issues discussions about font
support oral extensions HTTP additions and access related to fo RM
object title and language. The feature of ice add work phonetic
support at HP tml x, CSS, a discussion about soliciting ideas on
education and awareness plan and other programs in Europe. Do em slash
API work. math support and certification and reading. There are two
lists, there were two lists the working group list is for actively
participating in the design coding and working specifications. And the
second level says the ID group the interest group. For information
about the activity such as meeting agenda day take turns being made
results, but also and also provided for an exchange of ideas. and give
input to the working group. The sessions on the guidelines. Delete
sections. So, be web resources used to talk about the meeting about
the two groups is called welcome and agenda at the website is https
colon slash slash Li s t.wp. org slash archives slash public slash w
three c hyphen w AI hyphen w g a slash 91997 slash April June. Oh, h
er APR au n slash 0004 dot html. The reference for the agenda, w c
three w three C which is civility initiative in parentheses w AI in
parentheses first technical meeting is on WW, https colon slash slash
w three dot orgy slash domain slash references slash mind 70522 slash
agenda dot html and a link to the minutes is also enclosed on the
agenda website. Also during the meeting the director of hiring the
director was discussed and responsibilities related to the IPO and
funding process status was also discussed. A second meeting was
scheduled in Austin at MIT for August fifth. That would discuss
guidelines and education. There was a suggestion that w three c
technical leaders should. So the, so the meaning should run in
conjunction with other Ws three sync technical group meetings. So the
WEC technical leaders would be available on invitation for the
invitation for comments was suggested to be sent to the Wi Fi mailing
list. During the meeting, there were a specific a summary of the
current accessibility issues and considerations for the next release
of HTML code named Cougar co UTR and the sentence, starting at a
summary should be in quotes. The enhancements that were recommended
were labeled fields related fields with captions and access key
support tags and elements and Alp use were also addressed as they
relate to web accessibility issues. I need for both long and short
versions of the description was expressed. So that sighted visions,
people. So for example, sighted people may want a short description of
a graph, but a blind person may want additional information. Other
issues included content for both D tag and object elements requires
that the author's put in the content and the concern was whether the
author's would be motivated to do so. It turns out that the D tag is
an option. And that it met meets accessibility needs right now. I need
4441 name for CSS class name was identified and express the need to
discuss requirements before deciding on a solution was also mentioned
binding requirements, how to attach descriptions to tables, images and
sound, or what they use for what the content is what language to use
the urgency of, of, of editing these findings and a discussion about
what benefits that people and with disabilities, and both disabilities
and other people are there, such as you know what will benefit. What
will benefit people with disabilities that will also benefit other
people. Similar to the idea of sidewalk curb cuts and the plan was to
have I see a DD represented by Mike, and George k would be responsible
for posting the requirements to the working group list. There was also
an issue that the bindings not be intrusive. The issue that the title
attribute, not replace the detects was also me. The title attribute.
Basically it's captioning. I see a DD two two d t d was set up to
provide a structured documents solution. Quotes a beginning is
provide. So the publishers could create accessible documents for
students based on requirements originally specified in the Texas in
parentheses USA, close parentheses Braille bill. So, so I and I see a
DDS goal was to move toward a single DTD implement these HTML close
parentheses, for Accessible publishing, because the HTML is richer and
richer and far superior. So they should be quotes around, I see a DDS
goal from, I see a DDS goal to accessible publishing. No, no, no
quotes. Um, but this references w three.org slash w AI slash EMI NTT
dot html. And this page is entitled w AI technical meeting minutes.
Um, I see ABB status is related is handled by George cursher Capital
Ke Ke r s ch er suggestion to make Cougar Cougar clean. Pause part. I
so moved to make this happen. I'm not sure what Cougar coup is. There
were some IPP slash p elements in the ice add to, to, to HTML project.
So that Braille and navigation elements were added to include IPP and
I slash p elements as an anchor in HTML. To use meta email elements to
notify the browser that you're using this session. And there was a
recommendation as to reserve prefix space prefix namespace warrant
Lauren would talk about XML, which is a slimpar five version of sgml
elements using XML should start with XML underlying system, type
things. There was a suggestion that the Web Accessibility Initiative
technical working group should reserve the namespace by requesting it
to W ml wg. Let me stop you here. Enemy pick up a W three dot o r g
slash w AI slash m i n. Two Two dot html at a discussion of phonetic
support in HTML by Dave Ramsey.

**Comment by CherylLouise808** [comment]
Hi Eric:

Second interview-no response. Getting there. Condensing to word count.

Concerned that two interview absence will yield not per query.


Cheryl

**Comment by ericwbailey** [comment]
Thanks for the head's up. I've got a lot of trust for the end article's quality based on the draft and your communication, so please feel free to proceed with what you think is best.

**Comment by CherylLouise808** [comment]
Hi:

Thanks. That's pressure! May I do well.:-)

Cheryl

**Comment by CherylLouise808** [comment]
Hi Eric:

I am writing to let you know that the article is coming along. Just taking
time to sift through lots of notes and web references. No interviews have
panned out. It is not working out.

Soooo...in view of a late delivery of far from what I expected I cannot
THINK of sending an invoice, and let alone accepting payment.  We have our
in-house quality guidelines.

Please expect soon.

Cheryl

**Comment by ericwbailey** [comment]
Closed for inactivity.
